packages/e2e-tests/specs/experiments/multi-entity-editing.test.js

```diff
@@ -226,9 +233,11 @@ describe( 'Multi-entity editor states', () => {
 			removeErrorMocks();
 		} );
 
-		// Todo: Solve issue affecting test
-		// eslint-disable-next-line jest/no-disabled-tests
-		it.skip( 'should only dirty the parent entity when editing the parent', async () => {
+		it( 'should only dirty the parent entity when editing the parent', async () => {
+			// Clear selection so that the bock is not addded to template part.
```

Comment:
```suggestion
			// Clear selection so that the block is not added to the template part.
```

Reply:
Oops. Guess that’s what happens when I use my MacBook keyboard 🤷‍♂️